test/graphql/queries/profile_query_test.rb

```diff
@@ -12,6 +12,23 @@ class ProfileQueryTest < ActiveSupport::TestCase
       rule_count: 2,
       account: @user.account
     )
+
+    compliance_admin = RBACApiClient::AccessPagination.new(
+      data: [
+        RBACApiClient::Access.new(
+          permission: Rbac::COMPLIANCE_ADMIN,
+          resource_definitions: nil
+        ),
+        RBACApiClient::Access.new(
+          permission: Rbac::INVENTORY_VIEWER,
+          resource_definitions: nil
+        )
+      ]
+    )
+    RBACApiClient::AccessApi
+      .any_instance
+      .stubs(:get_principal_access)
+      .returns(compliance_admin)
```

Comment:
This could be moved into a parametrized helper function in the test helper somewhere.